zerver/tests/test_events.py

```diff
@@ -1389,7 +1389,7 @@ def do_set_user_display_settings_test(self, setting_name: str) -> None:
             raise AssertionError("Unexpected property type %s" % (property_type,))
 
         num_events = 1
-        if setting_name == "timezone":
+        if setting_name == "timezone" or setting_name == "emojiset":
```

Comment:
This could be better written as:
`if setting_name in ["emojiset", "timezone"]:`